qds_sdk/account.py

```diff
@@ -94,7 +94,7 @@ def branding(args):
         return result
 
 class Account(SingletonResource):
-    credentials_rest_entity_path = "accounts/get_creds"
+    credentials_rest_entity_path = "accounts/creds"
```

Comment:
This is relevant to the multicloud initiative. The get_creds api is replaced by creds api as a part of multicloud efforts.